### PR TITLE
Add new Header component and Icons to component library

### PR DESCRIPTION
**Pull request by eddiesarevalo**
## 🗒️ Summary
-Added Header component.
-Changed styling for components from JS in CSS to basic CSS, SASS and class names.
-Styles now use SASS variables to get common values.
-Added Icons.
-Added more instructions to readme.

## ⚙️ Test Data and/or Report
Follow the steps in the read me to build and run locally https://github.com/NASA-PDS/wds-react/edit/header/README.md#getting-started. The only difference is in step 1 use branch `header` for both repo's instead of `develop`. In summary download the wds and wds-react projects. Build them and then link them, after that build a test project to show the components.

## ♻️ Related Issues
-refs NASA-PDS/portal-wp-tasks/issues/55

### CODE REVIEW

File: src/components/TitleBar/TitleBar.scss
Comment (anilnatha) [inline src/components/TitleBar/TitleBar.scss:1]: Based on our conversation this morning, should these styles be added to `wds` instead and consumed by wds-react components?

File: src/components/Chip/Chip.scss
Comment (anilnatha) [inline src/components/Chip/Chip.scss:1]: Based on our conversation this morning, should these styles be added to `wds` instead and consumed by wds-react components?

File: src/components/Navbar/Navbar.scss
Comment (anilnatha) [inline src/components/Navbar/Navbar.scss:1]: Based on our conversation this morning, should these styles be added to `wds` instead and consumed by wds-react components?

File: src/components/Pagination/Pagination.scss
Comment (anilnatha) [inline src/components/Pagination/Pagination.scss:1]: Based on our conversation this morning, should these styles be added to `wds` instead and consumed by wds-react components?

File: src/components/Tag/Tag.scss
Comment (anilnatha) [inline src/components/Tag/Tag.scss:1]: Based on our conversation this morning, should these styles be added to `wds` instead and consumed by wds-react components?

File: src/components/Icons/ArrowCircleDown.tsx
Comment (anilnatha) [inline src/components/Icons/ArrowCircleDown.tsx:1]: @eddiesarevalo I couldn't find a reference to a template being used to generate these files. Is there a template file?  I did find the template file that is being used to generate the `index.ts` file for the list of Icons we export.

**Comment by eddiesarevalo** [comment]
@anilnatha The styles should not be moved to the wds because these styles are material ui specific for example this class:
`.pds-wds-titlebar-menu-item:hover .MuiListItemIcon-root .MuiListItemText-primary` is overriding material ui. Wds should be independent of material ui.

File: src/components/Icons/ArrowCircleDown.tsx
Comment (eddiesarevalo) [reply src/components/Icons/ArrowCircleDown.tsx:1]: @anilnatha We are using svgrrc. https://github.com/NASA-PDS/wds-react/blob/header/.svgrrc.cjs This uses the https://github.com/NASA-PDS/wds-react/blob/header/icon-index-template.cjs file to create the icons from the wds svg files.

**Comment by anilnatha** [comment]
> @anilnatha The styles should not be moved to the wds because these styles are material ui specific for example this class: `.pds-wds-titlebar-menu-item:hover .MuiListItemIcon-root .MuiListItemText-primary` is overriding material ui. Wds should be independent of material ui.

@eddiesarevalo Understood, I was worried about classes where a MUI style wasn't being overridden. I was of the thought that our `wds` repo would contain the base class for a given style definition from Figma. And in our `wds-react` repo we use that base class in our MUI components and add the needed overrides to handle the MUI style updates. Otherwise there is nothing in the `wds` repo that defines the style for a given component.

**Review by anilnatha — APPROVED** [review]

**Pull request merged**